### ISSUE TITLE
beginShape(POINTS) don't show up in P2D

### ISSUE THREAD

**Issue by clankill3r** [processing/processing#3029]
beginShape(POINTS) don't show up in P2D

JAVA2D and P3D work fine.

```java
PShape points;
 
void setup () {
 
  size(600, 600, P2D);
 
  points = createShape();
  points.beginShape(POINTS);
  points.strokeCap(SQUARE);
  for (int r=0; r<256; r++) {
    for (int g=0; g<256; g++) {
 
      points.stroke(color(r, g, 255));
      points.vertex(random(width), random(height));
    }
  }
  points.endShape();
}
 
 
void draw() {
 background(255);
 shape(points);
}
```

**Comment by JakubValtar** [comment]
Which version are you using?

In 3.0a5, I get the same output in both P2D and P3D. This seems to be a duplicate of #2912, fixed in October: #2915.

I just don't understand understand how it could work in JAVA2D. I get `createShape(), or this particular variation of it, is not available with this renderer.` in 2.2.1 and white screen in newer releases, which may be a different bug.

**Comment by codeanticode** [comment]
Fixed on 3.0a10.
